references/video_classification/train.py

```diff
@@ -149,7 +155,12 @@ def main(args):
     print("Loading validation data")
     cache_path = _get_cache_path(valdir)
 
-    transform_test = presets.VideoClassificationPresetEval((128, 171), (112, 112))
+    if not args.weights:
+        transform_test = presets.VideoClassificationPresetEval((128, 171), (112, 112))
+    else:
+        fn = PM.video.__dict__[args.model]
```

Comment:
are we providing some sort of registration API to get the models without having to resort to `__dict__` manipulations?

Reply:
Yeap, that's the plan. There will be a proper registration mechanism, possibly something similar to what was discussed [here](https://github.com/datumbox/dapi-model-versioning). There are still pending discussions with other domains, so I didn't want to adopt something before those discussions take place.